CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# How to contribute to OpenSpeedMonitor
+
+*Thank you for taking the time* :+1:
+
+There are two major ways in which you are able to contribute this project:
```

Comment:
Maybe we could boost the introduction a little bit more by enhancing that we welcome any contribution to the project.